src/themes/defaultTheme/sizes.js

```diff
@@ -23,16 +23,16 @@ const sizes = {
     h17: '34px',
     h18: '38px',
     h19: '42px',
-    h20: '46px',
-    h21: '50px',
-    h22: '60px',
-    h23: '70px',
-    h24: '80px',
-    h25: '90px',
-    h26: '100px',
-    h27: '110px',
-    h28: '120px',
-    h29: '130px',
+    h20: '44px',
```

Comment:
Are you sure about this ? you're breaking the incremental of 4px standard we set ?

Reply:
Totally breaking it ! But the standard is one thing, but having a good display is another thing. IMO I prefer having good looking components ;)